setup/package.dfdebian

```diff
@@ -51,7 +54,8 @@ RUN apt-get update -qq &&  \
         python3-vatnumber \
         python3-vobject \
         python3-werkzeug \
-        python3-xlsxwriter && \
+        python3-xlsxwriter \
+        rsync && \
```

Comment:
Why?

Reply:
because of this build depends https://github.com/odoo/odoo/blob/d46befdf25ad6a1f4c47a0308f2fbadfcfbcc6b7/debian/control#L6